charts/jaeger-v2/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "jaeger-v2.fullname" . }}
+  namespace: {{ template "jaeger-v2.namespace" . }}
+  labels:
+    {{- include "jaeger-v2.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "jaeger-v2.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "jaeger-v2.selectorLabels" . | nindent 8 }}
+        {{- include "jaeger-v2.podLabels" . | nindent 8}}
+    spec:
+      containers:
+        - name: jaeger-v2
+          image: {{ .Values.image }}
+          args: 
+            - "--config"
+            - "/etc/jaeger/config.yaml"
+          ports:
```

Comment:
missing the main OTLP ports

Reply:
what are the port number and name i should put

Reply:
all ports are defined here: https://github.com/jaegertracing/jaeger/blob/main/cmd/jaeger/Dockerfile